Compilation error

I am getting the following error when running `catkin_make`:

```
kuka-lwr/lwr_controllers/src/joint_impedance_controller.cpp:94:101: error:
no ‘void lwr_controllers::JointImpedanceController::commandConfiguration(const ConstPtr&)’
member function declared in class ‘lwr_controllers::JointImpedanceController’
void JointImpedanceController::commandConfiguration(const std_msgs::Float64MultiArray::ConstPtr &msg){
```

@marcoesposito1988 you renamed `JointImpedanceController::command` to `JointImpedanceController::commandConfiguration` in https://github.com/CentroEPiaggio/kuka-lwr/commit/7319f24d487e67862acc40a905540018092f01c6 without changing the header file

*Edit*: Actually `JointImpedanceController::command` is mentioned in other places also. @marcoesposito1988 why did you change it to commandConfiguration in `lwr_controllers/src/joint_impedance_controller.cpp`?

--- a/lwr_controllers/src/joint_impedance_controller.cpp
+++ b/lwr_controllers/src/joint_impedance_controller.cpp
@@ -91,7 +91,7 @@ void JointImpedanceController::update(const ros::Time& time, const ros::Duration
 }
 
 
-void JointImpedanceController::commandConfiguration(const std_msgs::Float64MultiArray::ConstPtr &msg){
+void JointImpedanceController::command(const std_msgs::Float64MultiArray::ConstPtr &msg){
   if (msg->data.size() == 0) {
     ROS_INFO("Desired configuration must be: %lu dimension", joint_handles_.size());
     }

